app/views/studios/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1>Studios</h1>
+<% @studios.each do |studio| %>
+  <%= studio.name %>
+  <%= studio.location %>
+    <% studio.movies.each do |movie| %>
+      <%= movie.title %>
+      <%= movie.creation_year %>
+      <%= movie.genre %>
+    <% end %>
+<% end %>
+
+<%# If i have time i can try to refactor lines 5-9 %>
```

Comment:
Don't forget that this is an HTML file and therefore you want to be using HTML to organize this file. I know you mentioned coming back to this file at a later time so I won't hold it against you here. Just keep in mind that this is an HTML file.